cmd/util_test.go

```diff
@@ -51,3 +50,11 @@ func TestCertificateResourceV1(t *testing.T) {
 	assert.Nil(result)
 	assert.NotNil(warn)
 }
+
+func TestCustomResourceDefinition(t *testing.T){
+	file := "../fixtures/custom_resource_definition.yml"
+	assert := assert.New(t)
+	resources, err := getKubeResourcesManifest(file)
+	assert.Nil(err)
+	assert.Len(resources, 1)
+}
```

Comment:
Please ensure you end all files with newline as many Unix tools don't behave properly with files not ending with newline chars.